languages/python/sqlalchemy-oso/sqlalchemy_oso/partial.py

```diff
@@ -16,41 +17,51 @@
 EmitFunction = Callable[[Session, Any], Any]
 
 
-def partial_to_filter(expression: Expression, session: Session, model):
+def partial_to_filter(expression: Expression, session: Session, model, get_model):
```

Comment:
Maybe thread some of these arguments around as `**kwargs`? Would make future changes to the arity less bigly.